### PR TITLE
Implement working data generation scripts and database seeding scripts

### PR DESCRIPTION
**Pull request by dicksonwmk**

### CODE REVIEW

File: database/cassandra/dataGenerationScripts/menuItemsGenerator.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      itemId++;
+      restaurantId++;
+      
+      if (restaurantId > 10000000) {
+        restaurantId = 1;
+      }
+    }
+
+    if (itemId < maxLimit) {
+      fil.once('drain', writer);
```
Comment (velezjose) [inline database/cassandra/dataGenerationScripts/menuItemsGenerator.js:64]: Great usage of `drain`!! How much faster do you think it is than writeFile or than the csv-writer npm module?

File: database/cassandra/schema.cql [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  description text,
+  pictureUrl text,
+  popular boolean,
+  spicy boolean,
+  required text,
+  optional text
+);
+
+COPY menu_items(itemId,restaurantId,category,name,price,description,pictureUrl,popular,spicy,required,optional) 
+    FROM '/Users/phdickson/Desktop/SDC/FEC-menu/database/cassandra/dataGenerationScripts/menuItemsCassandra.csv'
```
Comment (velezjose) [inline database/cassandra/schema.cql:21]: I think it's best practice if you hide the entire filepath to your `.csv` file and not make it public.

Comment (velezjose) [reply database/cassandra/schema.cql:21]: And instead write a `FILL_ME_IN`flag.

File: database/postgresql/schema.sql [hunk 3]
```diff
@@ -0,0 +1,53 @@
+-- If grubhub schema does not exist, create it
+CREATE SCHEMA IF NOT EXISTS grubhub AUTHORIZATION phdickson;
```
Comment (velezjose) [inline database/postgresql/schema.sql:2]: Again, don't think it's a good idea to make the `AUTHORIZATION` info public. A FILL_ME_IN can hide this.

**Review by velezjose — APPROVED** [review]
Good job implementing the seeding scripts and schemas for your dbs. Two quick suggestions for next time and for future PRs that I've found really helpful when reviewing other people's code: 

1. Writing a brief summary in the description of the PR stating the main things you did (being more specific than just having a title) is really helpful for the reviewer of your code to know what you did in what file and to have some context of where the state of the code was pre- and post- PR.

2. Writing more comments in your code can also be helpful for your code reviewer. 

Other than that, approved! Great job.

File: database/cassandra/schema.cql (hunk 2 above)
Comment (dicksonwmk) [reply database/cassandra/schema.cql:21]: Thanks for the tip. I will change that.

**Pull request merged**